compiler/src/iree/compiler/Codegen/Common/CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ iree_cc_library(
     "BufferizationAnalysis.h"
     "EncodingUtils.h"
     "ExtractAddressComputation.h"
+    "FastMathDeviceLib.h"
```

Comment:
you need to update the BUILD.bazel file (the cmake files are autogenerated, as noted at the top of the file)